### DISCUSSION TITLE
Cloud Key Gen2 firmware 2.0.24 and UniFi Controller v 6.0.43

### DISCUSSION THREAD

**Discussion by astro82** [Art-of-WiFi/UniFi-API-client#89] [Show and tell]
Ubiquiti released new firmware for Cloud Key Gen2 (v 2.0.24) which implements UniFi OS (Controller v 6.0.43). Port 8443 is not used anymore. Updated to latest versions of client and browser, erased port from address in config file and everything works like a charm!

Thank you so much for your great work!

**Comment by malle-pietje** [comment]
Thanks for the feedback!
